lib/main.dart

```diff
@@ -1,244 +1,19 @@
-                    mixedContentMode: AndroidMixedContentMode.MIXED_CONTENT_ALWAYS_ALLOW,
-                    thirdPartyCookiesEnabled: true,
-                    supportMultipleWindows: true,
-                  ),
-                ),
-                shouldInterceptFetchRequest: (controller, fetchRequest) {
-                  return Future.value(fetchRequest);
-                },
-                onCreateWindow: (controller, createWindowAction) {
-                  return Future.value(false);
-                },
-                onLoadResource: (controller, resource) {
-                  print(resource);
-                },
-                shouldInterceptAjaxRequest: (controller, ajaxRequest) {
-                  return Future.value(ajaxRequest);
-                },
-                shouldOverrideUrlLoading: (controller, navigationAction) {
-                  return Future.value(NavigationActionPolicy.ALLOW);
-                },
-                onLoadStop: (controller, url) async {
-                  if (url != null) {
-                    await updateCookies(url);
-                  }
-                },
-                onDownloadStart: (controller, url, userAgent,
-                    contentDisposition, mimeType, contentLength) async {
-                  void _download(String url) async {
-                    final status = await Permission.storage.request();
-
-                    if (status.isGranted) {
-                      String? localPath = await _findLocalPath();
-                      final id = await FlutterDownloader.enqueue(
-                        url: url,
-                        savedDir: localPath!,
-                        showNotification: true,
-                        headers: {
-                          HttpHeaders.cookieHeader: cookieString,
-                          HttpHeaders.contentTypeHeader: mimeType,
-                          HttpHeaders.connectionHeader: 'keep-alive',
-                          // 'Content-Length': contentLength.toString(),
-                          // 'User-Agent': userAgent
-                        },
-                        fileName: 'demo.xlsx',
-                        saveInPublicStorage: true,
-                        openFileFromNotification: true,
-                      );
-                    } else {
-                      print('Permission Denied');
-                    }
-                  }
-
-                  _download(_screenerHomeUrl + url.path);
-                },
-              ),
->>>>>>> 3424b2dffe6c79f6a5dc06e21a5908669fe9c32b
-            ),
-          ),
-        ));
-  }
-
-  Future<void> updateCookies(Uri url) async {
-    var cookies = await CookieManager().getCookies(url: url);
-    cookieString = "";
-    for (var cookie in cookies) {
-      cookieString += cookie.name + "=" + cookie.value;
-      if (cookies.indexOf(cookie) != cookies.length - 1) {
-        cookieString += "; ";
-      }
-    }
+      title: 'Screener',
+      home: Splash(debug: true),
```

Comment:
Debug to be false in production.

Reply:
Debug is set to false to test payments in local env, while pushing will switch to false